jira_projects_create_cards.py

```diff
@@ -0,0 +1,340 @@
+        match_projects = []
+
+        for project in projects:
+            project_name = self._format_project_name(project.name)
+            for repo in repo_list:
+                if project_name.startswith(repo):
+                    match_projects.append(project)
+        return match_projects
+
+    ### Formating methods ###
+    ##########################
+
+    def _format_repo_list(self, repo_list):
+        """
+            Formating github repository name.
+
+            :param repo_list: repository list
+            :return: repository list formated
+        """
+        suffixs = ['_odoo', '_openerp']
+        for repo in repo_list:
+            for suffix in suffixs:
+                repo = repo.replace(suffix, "")
+        return repo_list
+
+    def _format_project_name(self, project_name):
+        """ Formating jira project name.
+
+            :param project_name: jira project name
+            :return: repository jira project name formated
+        """
+        pjct_name_no_space = project_name.replace(" ", "")
+        return pjct_name_no_space.lower()
+
+    def _check_project_list(self, projects):
+        """ Edit interactively the list of projects.
+            Like that we are able to remove projects and add some projects
+
+            :param projects: jira project list
+            :return: repository jira project list updated by the user
+
+            # TODO: Maybe also output the ignored lines.
+        """
+        projects_names = [project.name for project in projects]
+        initial_message = '\n'.join(projects_names)
+        initial_message = "# Project list :\n" + \
+            initial_message + \
+            "\n# Remove some of them if you want"
+
+        with tempfile.NamedTemporaryFile(suffix=".tmp") as tf:
+            tf.write(initial_message.encode('ascii'))
+            tf.flush()
+            call([EDITOR, tf.name])
+
+            # do the parsing with `tf` using regular File operations.
+            # for instance:
+            tf.seek(0)
+            edited_message = tf.read()
+        # Decode and remove the comment
+        projects_names_updated = edited_message.decode(
+            'ascii'
+        ).split('\n')[1:-2]
+        # Keep only the project in the name retrived before
+        projects_list_updated = [
+            project for project
+            in projects if project.name in projects_names_updated
+        ]
+        return projects_list_updated
+
+    ### Creating issue content ###
+    ##############################
+
+    def get_csv_jira_card(self, csv_file):
+        """ Retrieve the different parts used to create new issues
+            in the csv file.
+
+            :param csv_file: csv file with column 'project'
+            :return: lite of dict to create one issue
+        """
+        issues_list = []
+        with open(csv_file, newline='') as file:
+            reader = csv.DictReader(file)
+            for row in reader:
+                issue_dict = {
```

Comment:
why do you need another dict if you are reading a dict and not changing the keys?